packages/flutter_tools/lib/src/commands/config.dart

```diff
@@ -20,6 +22,11 @@ class ConfigCommand extends FlutterCommand {
       help: 'Clear the saved development certificate choice used to sign apps for iOS device deployment.');
     argParser.addOption('android-sdk', help: 'The Android SDK directory.');
     argParser.addOption('android-studio-dir', help: 'The Android Studio install directory. If unset, flutter will search for valid installs at well-known locations.');
+    argParser.addOption('jdk-dir', help: 'The Java Development Kit (JDK) install/home directory. '
+      'If unset, flutter will search for one in the following order:\n'
+      '    1) the JDK bundled with the latest installation of Android Studio,\n'
+      '    2) the JDK found at the directory found in the JAVA_HOME environment variable, and\n'
+      "    3) the directory containing the java binary found in the user's path.");
```

Comment:
I'd like to get this ordered list to be indented, but I couldn't find a way to do this. Indenting with spaces or using `\t` doesn't seem to work as this seem to get removed in the output. (The spaces currently here are there to satisfy args_test.dart). Here is the output from `flutter config`:

```
    --android-sdk                               The Android SDK directory.
    --android-studio-dir                        The Android Studio install directory. If unset, flutter will search for valid installs at well-known locations.
    --jdk-dir                                   The Java Development Kit (JDK) install/home directory. If unset, flutter will search for one in the following order:
                                                1) the JDK bundled with the latest installation of Android Studio,
                                                2) the JDK found at the directory found in the JAVA_HOME environment variable, and
                                                3) the directory containing the java binary found in the user's path.
    --build-dir=<out/>                          The relative path to override a projects build directory.
    --[no-]enable-web                           Enable or disable Flutter for web.
```